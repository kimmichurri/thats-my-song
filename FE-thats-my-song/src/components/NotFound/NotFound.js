import React from 'react';

export default function NotFound() {
  return (
    <div className='not-found-wrapper'>
      <h3 className='not-found-text'>404 page not found</h3>
    </div>
  )
}