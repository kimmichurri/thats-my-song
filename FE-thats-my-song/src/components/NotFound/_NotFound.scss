.not-found-wrapper {
  display: flex;
  justify-content: center;
}

.not-found-text {
  font-size: 5rem;
  font-family: $accent-font;
}