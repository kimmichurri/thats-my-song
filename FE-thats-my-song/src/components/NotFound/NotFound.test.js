import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});