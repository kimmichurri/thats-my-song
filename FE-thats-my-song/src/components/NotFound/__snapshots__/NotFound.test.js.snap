// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match the snapshot 1`] = `
"<div className=\\"not-found-wrapper\\">
  <h3 className=\\"not-found-text\\">
    404 page not found
  </h3>
</div>"
`;
